src/components/HistoryCard/ViHistoryCard.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { useHistory, useAuth } from "../../context/context";
+import "./ViHistoryCard.css";
+import axios from "axios"
+
+
+const ViHistoryCard = ({ video }) => {
+  const {state: { encodedToken },} = useAuth();
+    const {dispatch} = useHistory();
+    
+    /** 
+     * @param {*} Removes Single video from history page 
+     */
```

Comment:
can be removed

Reply:
These are JS docs which you keep to make sure of viewers what functions you are using and what it returns